test/utils/ViewTest.scala

```diff
@@ -82,6 +82,9 @@ trait ViewTest extends UnitTest with GuiceOneAppPerSuite {
         s"has a href to '${href.get}'" in {
           document.select(selector).attr("href") shouldBe href.get
         }
+        s"has a role of button" in {
```

Comment:
if im going to be really picky 🙈, you dont need the s before s"has a role of button" because you're not passing anything to the string

Reply:
you're right, copy and paste gone wrong.